### PR TITLE
Avoid deadlock in Target initialization

### PR DESCRIPTION
**Pull request by thiagowfx**
When initialising new tab, don't wait for the result of the commands can be blocked by debugger.

Bug: #293

### CODE REVIEW

**Comment by sadym-chromium** [comment]
This is the way to create a new tab I told you about: `data:text/html,<a href="https://example.com" target="_blank">new tab</a>`

**Comment by thiagowfx** [comment]
> This is the way to create a new tab I told you about: data:text/html,<a href="https://example.com" target="_blank">new tab</a>

In this way, how do we force a click on the link? We don't have element selectors yet, right?

**Comment by thiagowfx** [comment]
How about https://github.com/GoogleChromeLabs/chromium-bidi/pull/752/commits/aad339b95c961a8de5fa85a476675188b0f79738?

File: tests/test_preload_script.py [hunk 1]
```diff
@@ -489,6 +490,48 @@ async def test_addPreloadScriptGlobally_loadedInNewContexts(
...
+                "target": {
+                    "context": context_id
+                },
+                "awaitPromise": True,
+                "resultOwnership": "root"
+            }
+        })
+    assert result["result"] == {"type": "string", "value": 'bar'}
+
+    # TODO: Click the link and assert that the new tab has the preload script.
```
Comment (sadym-chromium) [inline tests/test_preload_script.py:532]: You can use BiDi+ to send [`Runtime.evaluate`](https://chromedevtools.github.io/devtools-protocol/tot/Runtime/#method-evaluate) with `userGesture: true`.

Comment (thiagowfx) [reply tests/test_preload_script.py:532]:
```
    await send_JSON_command(
        websocket, {
            "method": "cdp.sendCommand",
            "params": {
                "cdpMethod": "Runtime.evaluate",
                "cdpParams": {
                    "expression": "document.querySelector('a').click();",
                    "userGesture": True
                },
                "cdpSession": cdp_session
            }
        })
    await read_JSON_message(websocket)
```

Is this what you mean?

Comment (thiagowfx) [reply tests/test_preload_script.py:556]: https://github.com/GoogleChromeLabs/chromium-bidi/pull/752/commits/4dc21c694f821cfe708b7912170f6da7b51a78fd

Comment (thiagowfx) [reply tests/test_preload_script.py:556]: Should we just add a TODO here for now and skip it as you previously suggested?

Comment (sadym-chromium) [reply tests/test_preload_script.py:556]: if it does not work, you can skip test for now, and leave a todo

Comment (sadym-chromium) [reply tests/test_preload_script.py:556]: did you try running it manually or in headful mode?

Comment (sadym-chromium) [reply tests/test_preload_script.py]: https://github.com/GoogleChromeLabs/chromium-bidi/pull/752/commits/3c8d2c5e1b48760630ab3ea29fd7db12ea6eb696

**Comment by thiagowfx** [comment]
Old: https://github.com/GoogleChromeLabs/chromium-bidi/pull/772

@sadym-chromium Rebased, feel free to take this over.

**Comment by thiagowfx** [comment]
The `log_entry_added` is never emitted. The test hangs.

**Review by sadym-chromium — APPROVED** [review]

**Pull request merged**